src/query/storage/promremote/query_converter_test.go

```diff
@@ -0,0 +1,80 @@
+package promremote
+
+import (
+	"testing"
+	"time"
+
+	"github.com/prometheus/prometheus/prompb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/m3db/m3/src/query/models"
+	"github.com/m3db/m3/src/query/storage"
+	"github.com/m3db/m3/src/query/ts"
+	xtime "github.com/m3db/m3/src/x/time"
+)
+
+func TestQueryConverter(t *testing.T) {
```

Comment:
Nit (I find the usage of "Query" for writes ver confusing in general):
```suggestion
func TestWriteQueryConverter(t *testing.T) {
```